gradle/dependencies.gradle

```diff
@@ -29,7 +29,7 @@ ext {
 }
 
 // Add Scala version
-def defaultScala212Version = '2.12.10'
+def defaultScala212Version = '2.12.11'
```

Comment:
we should also update `gradle.properties`, `bin/kafka-run-class.sh`, `bin/windows/kafka-run-class.bat`?

Reply:
Yes.